CMakeLists.txt

```diff
@@ -728,6 +728,18 @@ if(FLB_BACKTRACE)
   include_directories("${CMAKE_CURRENT_BINARY_DIR}/backtrace-prefix/include/")
 endif()
 
+if(FLB_OUT_KAFKA)
+    FLB_OPTION(RDKAFKA_BUILD_STATIC    On)
+    FLB_OPTION(RDKAFKA_BUILD_EXAMPLES Off)
+    FLB_OPTION(RDKAFKA_BUILD_TESTS    Off)
+    FLB_OPTION(ENABLE_LZ4_EXT         Off)
+
+    add_subdirectory(lib/librdkafka-1.7.0 EXCLUDE_FROM_ALL)
```

Comment:
Please use variable paths for the subdirectory and headers, e.g:

- https://github.com/fluent/fluent-bit/blob/master/cmake/libraries.cmake
- https://github.com/fluent/fluent-bit/blob/master/cmake/headers.cmake